app/vmalert/README.md

```diff
@@ -131,6 +131,10 @@ name: <string>
 # By default, "prometheus" type is used.
 [ type: <string> ]
 
+# Optional
+# Group evaluation time will always be aligned with interval.
```

Comment:
we should explain the following:
1. the `timestamp`, not time will be aligned
2. why alignment is needed

Although, why it is named `query_time_alignment` if we in `eval_offset` we refer to `evaluation`?

Reply:
Got it, fixed!
Used `query_time_alignment` to follow the old flag, now using `eval_alignment` to be align with `eval_offset`.